GameBoard/Areas/Identity/Pages/Account/Register.cshtml.cs

```diff
@@ -51,18 +43,14 @@ public async Task<IActionResult> OnPostAsync(string returnUrl = null)
 
                     var code = await _userManager.GenerateEmailConfirmationTokenAsync(user);
                     var callbackUrl = Url.Page(
-                        "/Account/ConfirmEmail",
+                        "/Account/ConfirmedEmail",
                         null,
                         new {userId = user.Id, code},
                         Request.Scheme);
 
-                    await _emailSender.SendEmailAsync(
-                        Input.Email,
-                        "Confirm your email",
-                        $"Please confirm your account by <a href='{HtmlEncoder.Default.Encode(callbackUrl)}'>clicking here</a>.");
+                    await _mailSender.SendEmailConfirmationAsync(Input.Email, HtmlEncoder.Default.Encode(callbackUrl));
```

Comment:
You can extract a method that's responsible for mail sending.

Reply:
Done

Reply:
Same as with `ForgotPassword`, top-down ordering.

Reply:
Done